gen-kubectldocs/config/v1_17/toc.yaml

```diff
@@ -0,0 +1,61 @@
+categories:
+- name: GETTING STARTED
```

Comment:
Around content organization, perhaps a grouping could be `APPLICATION MANAGEMENT`?

Going even further, I'd prefer to have groupings for `IMPERATIVE APPLICATION MANAGEMENT` and `DECLARATIVE APPLICATION MANAGEMENT` to mirror the corresponding doc sections under [Manage Kubernetes Objects](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/declarative-config/):

```
Declarative Management of Kubernetes Objects Using Configuration Files
Declarative Management of Kubernetes Objects Using Kustomize
Managing Kubernetes Objects Using Imperative Commands
Imperative Management of Kubernetes Objects Using Configuration Files
```

Reply:
@julianvmodesto, Should the group names be restricted to a specific length? The names are part of the left navigation.

Is it possible for `kubectl` to provide the groupings or categories. The toc could be dynamically generated instead of manually updated whenever the commands change.
I think there is an issue/PR in spf13/cobra requesting something like this (https://github.com/spf13/cobra/issues/836).